ARDOPC/Makefile

```diff
@@ -1,13 +1,72 @@
 #	ardopcf Makefile
+#		For Linux, the default build requires only gcc and libraries typically
+#		installed by default.
+#			cd ardop/ARDOPC
+#			make
+#
+#		Fow Windows, the default build requires installation of a MinGW build
+#		environment.  The easily installable packages from https://winlibs.com
+#		available for 32-bit or 64-bit builds are suggested.  These are used to
+#		build the Windows releases.  Installing these in `C:\Program Files` is
+#		not recommeneded since that may require admin privleges.  Other build
+#		environments may also work but are not tested.
+#			cd ardop\ARDOPC
+#			mingw32-make
+#
+#	`make test` which builds the executable and also runs some tests also
+#	requires installation of cmocka, which is not required for the default build.
+#		On Debian/Ubuntu this is easily installed with:
+#			sudo apt install libcmocka-dev
+#
+#		Package managers for other Linux distributions are also likely to
+#		provide easy installation of cmocka.
+#
+#		In the following description of how to install cmocka for Windows, a
+#		winlibs MinGW installation is assumed to be located at `C:\winlibs-32`
+#		or `C:\winlibs-64`.  If installed elsewhere, substitute the appropriate
+#		paths.  Putting the cmocka files into the winlibs install directory
+#		avoids the need for further configuration.
+#
+#		For Windows 32-bit:
```

Comment:
I'm not much of a Windows user any more either.  However, I have a Windows laptop that I use occasionally for things that are not convenient to do with Linux.  Now, I'm also using this Windows machine to build and test the Windows versions of ardopcf.  

dumpbin is part of Visual Studio, which I don't have installed.  However, another answer on the page you linked to pointed me to sigcheck, which is available as a standalone download.  This confirms that cmocka.dll from https://cmocka.org/files/1.1/cmocka-1.1.0-mingw.zip is for 32-bit.

I don't know how many 32-bit Windows installations are still out there.  Download statistics for ardopcf releases show about 4 times as many downloads of the Windows_64 version as the Windows_32 version.  I included the 32-bit version with my first release of ardopcf because I wondered whether anyone would want it.  Since it had some downloads, I've continued to provide them.

Reply:
Looking at this file list, it appears that they have not provided Windows binaries since v1.1.0 in 2016. The most current release is v1.1.7 in 2023. I would recommend building from source instead for both 32- and 64-bit targets.

Reply:
My initial attempt to build cmocka for 32-bit windows failed.  I'll explore this further to see if I can figure out why and how to fix it.